dist.ini

```diff
@@ -0,0 +1,74 @@
+name    = WebService-Async-SmartyStreets
+author  = Tom Molesworth <TEAM@cpan.org>
+license = Perl_5
+copyright_holder = Tom Molesworth
+copyright_year   = 2019
+main_module	= lib/WebService/Async/SmartyStreets.pm
+
+[Git::GatherDir]
+exclude_filename = Makefile.PL
+include_dotfiles = 1
+[PruneCruft]
+except = t/rc/\.perl.*rc$
+[ManifestSkip]
+[MetaYAML]
+[License]
+[Readme]
+[MakeMaker]
+eumm_version = 6.48
+prereq_fatal = 1
+[ExecDir]
+[ShareDir]
+dir = share
+[Manifest]
+[TestRelease]
+[ConfirmRelease]
+[Prereqs::FromCPANfile]
+[Prereqs / BuildRequires]
+perl = 5.014000
+[CheckPrereqsIndexed]
+[CheckExtraTests]
+[VersionFromModule]
+[OurPkgVersion]
+[InstallGuide]
+[MetaJSON]
+[InsertExample]
+[PerlTidy]
+perltidyrc = t/rc/.perltidyrc
+[PodSyntaxTests]
+[MojibakeTests]
+[Test::CheckDeps]
+[Test::Compile]
+[Test::Synopsis]
+; [Test::EOL]
+[Test::Version]
+[Test::Pod::LinkCheck]
+; [PodCoverageTests]
+[Test::UnusedVars]
+[Test::Perl::Critic]
+critic_config = /home/git/regentmarkets/cpan/rc/.perlcriticrc
+[Test::ReportPrereqs]
+[SpellingCommonMistakesTests]
+; [Signature]
+[CopyFilesFromRelease]
+filename = lib/LandingCompany.pm
```

Comment:
looks like everything in this `dist.ini` file was copied from one of our private modules - look in `binary-com` instead